src/Factories/FunFunFactory.php

```diff
@@ -1562,6 +1568,10 @@ public function getPaymentTypesSettings(): PaymentTypesSettings {
 		return $this->pimple['payment-types-settings'];
 	}
 
+	public function getPresetAmountsSettings(): array {
+		return $this->config['preset-amounts']['donation'];
```

Comment:
If we need to convert the amounts to `Euro` instances this would be a good spot - with all the consequences e.g. in `IsCustomAmountValidator`, but we'll never have to think about it again. It caught up on us quickly that we hoped we would get away with a native data type...